### PR TITLE
workflows

### PR DESCRIPTION
**Pull request by jesusfcr**
Create two reusable workflows to approve dependabot prs and release dependabot updates

### CODE REVIEW

File: .github/workflows/approve-dependabot-pr.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        required: true
+    inputs:
+      runs-on:
+        required: false
+        type: string
+        default: ubuntu-latest
+
+jobs:
+  approve:
+    runs-on: ${{ inputs.runs-on }}
```
Comment (manelmontilla) [inline .github/workflows/approve-dependabot-pr.yml:17]: Just curios (I don't know very well how reusable workflows work), what's the reason for defining and `runs-on` input? I mean, couldn't we just set it to `ubuntu-latest`?

**Review by manelmontilla — APPROVED** [review]
LGTM

File: .github/workflows/approve-dependabot-pr.yml (hunk 1 above)
Comment (jesusfcr) [reply .github/workflows/approve-dependabot-pr.yml:17]: Well, the problem is that our GHE we don't have `ubuntu-latest` as a valid runner.
Adding this parameter allows to use the workflow in that case by providing one available runner name.

Comment (manelmontilla) [reply .github/workflows/approve-dependabot-pr.yml:17]: Cool I didn't know, thanks for the clarification.

**Pull request merged**